ktor-io/jvm/test/io/ktor/utils/io/ByteBufferChannelTest.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.utils.io
+
+import kotlinx.coroutines.*
+import java.io.*
+import kotlin.test.*
+
+class ByteBufferChannelTest {
```

Comment:
Consider adding WebSocket tests with a cancellation

Reply:
Not sure how it relates to the unwrapping of cancellation exception, but I added `WebSocketTest.testCancel`, please have a look.